### PR TITLE
Added information about service metrics

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Added information about the Metrics tab for services.

### CODE REVIEW

File: docs/platform/howto/service-metrics.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   * **Day:** Metrics for the last day with a granularity of 5 minutes.
+   * **Week:** Metrics for the last week with a granularity of 30 minutes.
+   * **Month:** Metrics for the last month with a granularity of 3 hours.
+   * **Year:** Metrics for the last year with a granularity of 1 day.
+
+.. note:: 
+    The selected period is relative to the current date and time. For example, selecting the **Hour** option will retrieve metrics for the last hour.
+
+
+To further enhance your :doc:`monitoring capabilities </docs/platform/howto/monitoring-services>`, you can **Enable metrics integration** and establish a connection to push service metrics to an M3, InfluxDB®, or PostgreSQL® service within the Aiven platform. This integration allows you to conveniently send your service metrics to an existing service or create a new one dedicated to receiving and storing the metrics.
```
Comment (wojcik-dorota) [inline docs/platform/howto/service-metrics.rst:38]: 1. "establish a SERVICE connection"?
2. "or create a new one for receiving and storing the metrics"?

File: docs/platform/howto/service-metrics.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Access service metrics
+=======================
+
+Service metrics are essential for monitoring and evaluating how well your services are performing. They give you valuable information about how your services use resources, their efficiency, and their overall health. By tracking and analyzing service metrics, you can make data-driven decisions, identify potential issues, and optimize the performance of your services.
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/platform/howto/service-metrics.rst:4]: fixed.

File: docs/platform/howto/service-metrics.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+=======================
+
+Service metrics are essential for monitoring and evaluating how well your services are performing. They give you valuable information about how your services use resources, their efficiency, and their overall health. By tracking and analyzing service metrics, you can make data-driven decisions, identify potential issues, and optimize the performance of your services. 
+
+Available service metrics
+--------------------------
+
+The service metrics available in the Aiven Console include:
+
+* **CPU usage:** Indicates the percentage of CPU resources consumed by the service.
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/platform/howto/service-metrics.rst:11]: I will keep ":", as it helps to visually separate the metric name from its explanation, providing clarity and improving readability.

File: docs/platform/howto/service-metrics.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/platform/howto/service-metrics.rst:38]: I will keep this as is; the flow is self-explanatory when clicking the Enable metrics integration.

File: docs/platform/howto/service-metrics.rst [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+3. Choose the desired period for which you want to retrieve the metrics. The available options are:
+
+   * **Hour:** Metrics for the last hour with a granularity of 30 seconds.
+   * **Day:** Metrics for the last day with a granularity of 5 minutes.
+   * **Week:** Metrics for the last week with a granularity of 30 minutes.
+   * **Month:** Metrics for the last month with a granularity of 3 hours.
+   * **Year:** Metrics for the last year with a granularity of 1 day.
+
+.. note:: 
+    The selected period is relative to the current date and time. For example, selecting the **Hour** option will retrieve metrics for the last hour.
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/platform/howto/service-metrics.rst:35]: Fixed.

File: docs/platform/howto/service-metrics.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* **Disk space usage:** Represents the percentage of disk space utilized by the service.
+* **Disk iops (reads):** Indicates the input/output operations per second (IOPS) for disk reads.
+* **Disk iops (writes):** Indicates the input/output operations per second (IOPS) for disk writes.
+* **Load average:** Shows the 5-minute average CPU load, indicating the system's computational load.
+* **Memory usage:** Represents the percentage of memory resources utilized by the service.
+* **Network received:** Indicates the amount of network traffic received by the service, measured in bytes per second.
+* **Network transmitted:** Indicates the amount of network traffic transmitted by the service, also measured in bytes per second.
+
+View service metrics
+---------------------
```
[the remark this answers is not in the document]
Comment (harshini-rangaswamy) [reply docs/platform/howto/service-metrics.rst:21]: fixed

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**